src/zjs_timers.c

```diff
@@ -131,7 +140,7 @@ static jerry_value_t add_timer_helper(const jerry_value_t function_obj,
     zjs_timer_t* handle = add_timer(interval, callback, this, repeat, argv, argc - 2);
     if (handle->callback_id == -1)
         return zjs_error("native_set_interval_handler: timer alloc failed");
-    jerry_set_object_native_handle(timer_obj, (uintptr_t)handle, zjs_timer_free_cb);
```

Comment:
If I give JrS the ability to GC the timer it will do so if you don't assign the return value of setInterval. The other option was to just keep the previous change I made, allow JrS to handle the GC of timers, and require users always save the return value of setInterval in their JavaScript. i.e. use - var timers = setInterval(myFunc, 1000);  At least as far as I know, there's no way for this code to know the GC is intended or not.